proto/cluster.proto

```diff
@@ -144,6 +144,30 @@ message EnvironmentVariables {
   map<string, EnvValueFrom> valuesFrom = 2;
 }
 
+message AutoscalerOptions {
+  // IdleTimeoutSeconds is the number of seconds to wait before scaling down a worker pod which is not using Ray resources.
+	// Defaults to 60 (one minute).
+	int32 idleTimeoutSeconds = 1;
```

Comment:
Is this field required? If yes, let's adnotated as such.

Reply:
nop, none of the fields are